ocaml/src/asd_server.ml

```diff
@@ -1272,9 +1304,13 @@ let run_server
   in
 
   Lwt_log.info_f "asd_server version:%s" Alba_version.git_revision     >>= fun () ->
-  Lwt_log.debug_f "tls:%s" ([%show: Asd_config.Config.tls option] tls) >>= fun () ->
-
-  let ctx = Tls.to_server_context tls in
+  Lwt_log.debug_f "tls:%s" ([%show: Asd_config.Config.tls option] tls_config) >>= fun () ->
+  Lwt_log.debug_f "transport:%s" ([%show: Net_fd.transport] transport) >>= fun () ->
+  let tls =
+    match tls_config with
+    | None -> (None: Tls.t option)
+    | Some tls_config -> failwith "todo"
```

Comment:
what exactly is this todo about? you only mean to throw something when the (invalid) combination of rdma+tls is chosen?
also tls no longer works for the asds?
(probably I'm missing something...)